### ISSUE TITLE
Keyboard shortcut for loading a session

### ISSUE THREAD

**Issue by manucontrovento** [openAVproductions/openAV-Luppp#291]
Hello everyone, 

I think that a quick, handy "load session" key or midi binding could be useful...to quickly load a session without using the mouse. Think about using "F12" of "CTRL+O" or "\"... or "*"... something handy to quickly recall the sessions list

Bye
Manu

**Comment by georgkrause** [comment]
There is #236, maybe thats a better fit for your problem?

**Comment by manucontrovento** [comment]
Yes, something to quickly load a song... with pc keyboard or midi shortcuts. 
Maybe from a playlist.
Something which would avoid using the mouse, kkeping your hands on keyboard and MIDI controller.
